pages/data-providers/[data-provider-id]/doi.jsx

```diff
@@ -107,7 +107,7 @@ const DepositDates = ({
         searchable
       >
         <Table.Column
-          id="repo_doi"
+          id="repoDoi"
```

Comment:
Is `id` used in any way?
- If is, why?
- If it's not, can it be just `doi` or `origin-doi`?

Reply:
It's used for sorting. The value is appended after sortBy parameter. Also it's used as a default getter for cell value.